R/download_dataset.R

```diff
@@ -0,0 +1,170 @@
+#' Download data for publication
+#'
+#' ...
```

Comment:
Could you maybe write one or two sentences about it?
Just an idea: _This function allows to download data as csv files. The data are downloaded as they are in database, i.e. no consistency check is performed. We recommend to check them by .. before publication._

Reply:
Yes, that part wasn't finished yet indeed!

Reply:
Now documented